add_me_to_your_PATH/julia_pod

```diff
@@ -128,14 +128,18 @@ fi
 # pod will fail to start unless secret and key are present. When `GITHUB_TOKEN_FILE` isn't set this will be populated with ""
 echo "GITHUB_TOKEN=$([ -z ${GITHUB_TOKEN_FILE} ] || cat ${GITHUB_TOKEN_FILE})" | kubectl create secret generic $SECRET_NAME --from-env-file=/dev/stdin
 
+#GIT_INFO="${GIT_REPO}_${GIT_BRANCH}"
+
 # substitute ENV vars into driver.yaml.template > driver.yaml for this run
 log "Generating $DRIVER_YAML" \
     && IMAGE_TAG=$IMAGE_TAG \
         RUNID=$RUNID \
         KUBERNETES_SERVICEACCOUNT=$KUBERNETES_SERVICEACCOUNT \
         JULIA_COMMAND=$JULIA_COMMAND \
         SECRET_NAME=$SECRET_NAME \
-            envsubst '${IMAGE_TAG} ${RUNID} ${KUBERNETES_SERVICEACCOUNT} ${JULIA_COMMAND} ${SECRET_NAME}' <driver.yaml.template >$DRIVER_YAML
+        GIT_REPO=$GIT_REPO \
+        GIT_BRANCH=$GIT_BRANCH \
+            envsubst '${IMAGE_TAG} ${RUNID} ${KUBERNETES_SERVICEACCOUNT} ${JULIA_COMMAND} ${SECRET_NAME} ${GIT_REPO} ${GIT_BRANCH}' <driver.yaml.template >$DRIVER_YAML
```

Comment:
FYI a cleaner way we could do this would be to use `env -i` and avoid having to specify all of the variables `envsubst` should use

Reply:
Due to `env -i` stripping out `PATH` you'll need to do `env -i PATH=$PATH ... envsubst <driver.yaml.template >$DRIVER_YAML` or `env -i ... $(command -v envsubst) <driver.yaml.template >$DRIVER_YAML`. The later expands the `PATH` prior to running `env -i`